cloud/blockstore/libs/storage/service/volume_session_actor_mount.cpp

```diff
@@ -689,6 +704,11 @@ void TMountRequestActor::HandleWaitReadyResponse(
 {
     Error = ev->Get()->Record.GetError();
 
```

Comment:
Maybe we should do this in HandleStartVolumeResponse, not in HandleWaitReadyResponse?

Reply:
You should request volume start just after it stopped.

Reply:
done